code/game/objects/structures/plasticflaps.dm

```diff
@@ -7,6 +7,17 @@
 	anchored = 1
 	layer = 4
 
+/obj/structure/plasticflaps/CanAStarPass(ID, to_dir, caller)
+	. = 1 //diseases, stings, etc can pass
+
+	if(istype(caller, /mob/living))
+		if(istype(caller,/mob/living/simple_animal/bot/mulebot))
```

Comment:
>. = 1
[some_code]
return

Return 1 in my testing (or whatever is in ```.```).
Anyway, if that's superflous, i'll revert to plain old return if you want.